### PR TITLE
{rdbms} Fix #17579: Add document reference where updated versions are…

### LINKED ISSUE #17579: no versions mentioned, old deprecated version installed if omitted
No examples are given for `--version`, and when omitted an old deprecated version is installed: 0.9.6

Please upgrade the default version and mention in the docs what is the default, and what versions are available.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d11bd3f4-efbf-691d-76a4-293ab97ad845
* Version Independent ID: a553d088-d838-f90f-d601-12fa312fe0cf
* Content: [az postgres server](https://docs.microsoft.com/en-us/cli/azure/postgres/server?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/postgres/server.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/postgres/server.yml)
* Service: **postgresql**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by RakeshMohanMSFT**
… available

Fixes https://github.com/Azure/azure-cli/issues/17579

**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
rdbms

**Review by evelyn-ys — APPROVED** [review]

**Review by alanenriqueo — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/rdbms/_params.py [hunk 1]
```diff
@@ -89,7 +89,11 @@ def _complex_params(command_group):  # pylint: disable=too-many-statements
             c.argument('assign_identity', options_list=['--assign-identity'], help='Generate and assign an Azure Active Directory Identity for this server for use with key management services like Azure KeyVault.')
 
             c.argument('location', arg_type=get_location_type(self.cli_ctx))
-            c.argument('version', help='Server major version.')
+            if command_group == 'postgres':
+                c.argument('version', default=11,
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/rdbms/_params.py:93]: The version seems to expect a `str` value but you set a `int` value as default

Comment (RakeshMohanMSFT) [reply src/azure-cli/azure/cli/command_modules/rdbms/_params.py]: done

**Comment by DaeunYim** [comment]
@alanenriqueo @RakeshMohanMSFT Let's change the default value for PG major version to be 13.

**Pull request merged**